src/Tribe/Linked_Posts/Base.php

```diff
@@ -339,7 +339,7 @@ public function find_with_upcoming_events( $only_with_upcoming = true, $event_po
 	 * @since TDB
 	 */
 	protected function prefix_key( $key ) {
-		if ( 0 !== strpos( $key, $this->meta_prefix ) && in_array( $key, Tribe__Events__Organizer::$meta_keys ) ) {
+		if ( 0 !== strpos( $key, $this->meta_prefix ) && in_array( $key, $this::$meta_keys ) ) {
```

Comment:
Add `true` as third argument to the `in_arrray` function to indicate it should be a strict comparison:
```suggestion
		if ( 0 !== strpos( $key, $this->meta_prefix ) && in_array( $key, $this::$meta_keys, true ) ) {
```
This will avoid the internal type casting and provide a small performance boost.